core/pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [project]
 name = "harambe-core"
-version = "0.51.0"
+version = "0.51.1"
```

Comment:
you'll need to run uv sync in this folder and also update the version in the sdk folder (and run uv sync)